legacyman_parser/process_regions.py

```diff
@@ -3,9 +3,12 @@
 from crawler.simple_crawler import SimpleCrawler
 
 
-def process_soup(soup: BeautifulSoup, url: str, parent_url: str):
-    print("Tango " + url + " " + str(parent_url))
+def process_soup(returned_soup: BeautifulSoup, url: str, parent_url: str):
+    """This assumes that there's going to be only
```

Comment:
that's not a valid assumption.  There are other tables, one contains DreamWeaver metadata and are hidden, another is a list of quick-links.

For the `pd_1.html` page, it's actually the `map` that you parse.

Reply:
>For the pd_1.html page, it's actually the map that you parse.

Okay. So, there's only one map, and we'll use it to parse regions.